app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +52,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+
+.warning {
+  color: $red;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: $white;
+  border: 1px solid transparent;
+  border-color: #BCE8F1;
+  border-radius: 4px;
+  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.05);
+}
+
+.card_edit {
+  border-color: $dark-blue;
+}
+
+.card__heading {
+  color:$white;
+  padding: 10px 15px;
+  border-bottom: 1px solid transparent;
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  background-color: $blue;
+  border-color: $blue;
+}
+
+.card__heading_edit {
+  background-color: $dark_blue;

```

Comment:
Name of variable `dark_blue` should be written in all lowercase letters with hyphens instead of underscores